README.md

```diff
@@ -0,0 +1,82 @@
+FAGC Discord Bot
+---
+
+## Contents
+- [FAGC Discord Bot](#fagc-discord-bot)
+- [Contents](#contents)
+- [Explanation](#explanation)
+  - [Key terms](#key-terms)
+  - [Data Types](#data-types)
+- [Installation](#installation)
+
+## Explanation
+
+[⬆️ Head back up](#contents)
+
+FAGC (Factorio Anti-Grief Community) is a community of people who try to work against griefers in Factorio.
+
+### Key terms
+
+[⬆️ Head back up](#contents)
+
+- Community: A community is a group of people that play Factorio together and use this API in a way to prevent griefers on their servers
+- Rule: A single item that describes something that must be followed, rules are categorized by which aspect of gameplay/user interaction it governances.
+- Report: A single instance of a single rule which has been broken (or held up). Can be revoked and then turn into a revocation
+- Profile: A collection of reports. Profiles are only one per player per community (a community can have multiple profiles for different players and a player can have multiple Profiles in different communities)
+- Revocation: When a community removes any profiles from a user. Revocations can never be turned back to reports
+
+### Data Types
+
+[⬆️ Head back up](#contents)
+
+This is a short description of the different data that you may encounter with this bot or it's ecosystem, it is however not a complete developer guide.
+All `id` properties are generated automatically by the API and cannot be set by users.
+- **Community**
+  - `id` - ID of the community. Used to fetch it only
+  - `name` - Name of the community. Does not need to be the same as the name of the community's guild server
+  - `contact` - Discord UserID of the contact user for the community
+  - `guildid` - Discord GuildID of the Discord guild for the community
+- **Report**
+  - `id` - ID of the report. Can be used to fetch it and revoke it (only if the report has been created by your community)
+  - `playername` - Name of the player that is being reported
+  - `adminId` - Discord UserID of the administrator that created this report
+  - `proof` - Proof of the report
+  - `description` - Description of the report
+  - `automated` - Whether the rule is automated or not
```

Comment:
To me, it appears that all reports must be submitted by a human. I'm therefore struggling to see the purpose of the automated flag - does it mean that the trigger for the report may have not been witnessed by a human?
I think this needs clarification here

Reply:
I don't actually know, this is there for legacy purposes. Was there on the original API and didn't require any special treatment so I kept it in. @Cooldude2606, do you think that reports should even have the `automated` flag?

Reply:
The `automated` flag was planned as a way to form a network of trust back when only banlist syncs were considered. When a player was banned it would be reported to FAGC. If another comuntiy then banned the player because they were reported on FAGC, they would also need to report the player as banned but use the automated tag. You could then filter communties based on a level of trust, ie allows only manauls bans or all bans. However, this was never discussed further, replaced in favour of rules, and is out of scope for this PR.